internal/utils/common.go

```diff
@@ -108,16 +108,20 @@ func GetTarget(dryRun bool) (string, string) {
 		return "fake", label
 	}
 
-	img = ReadCMDLineArg("cos-img/filename=")[0]
+	imgs := ReadCMDLineArg("cos-img/filename=")
 
 	// If no image just panic here, we cannot longer continue
-	if img == "" {
-		log.Logger.Fatal().Msg("Could not get the image name from cmdline (i.e. cos-img/filename=/cOS/active.img)")
+	if len(imgs) == 0 {
+		if IsUKI() {
+			imgs = []string{""}
```

Comment:
small nit: this seems to be just forced to return an empty string later on?

Reply:
yeah this is done as a workaround to use the same state all across the different paths. I could work around this tbh but was the easiet way of fixing this for now.